sdv/multi_table/base.py

```diff
@@ -126,21 +128,39 @@ def get_table_parameters(self, table_name):
                 A dictionary representing the parameters that will be used to instantiate the
                 table's synthesizer.
         """
+        future_warning_msg = (
+            'The get_table_parameters method is deprecated. Use the get_parameters method '
+            'to get synthesizer parameters for all tables.'
+        )
+        warnings.warn(future_warning_msg, FutureWarning)
+
         return self._table_parameters.get(table_name, {})
 
-    def get_parameters(self, table_name):
-        """Return the parameters used to instantiate the table's synthesizer.
-
-        Args:
-            table_name (str):
-                Table name for which the parameters should be retrieved.
+    def get_parameters(self):
```

Comment:
Just curious, would we want to still be able to get single table parameters? Maybe leave `table_name = None` to get all tables otherwise get the specified one?

Reply:
Talked to @npatki about this a bit, and the preferred solution (if we want this functionality) would be to update `get_table_parameters` to do this instead of adding an argument here.